section-6/exercises/OSidorov_Sec_6.js

```diff
@@ -0,0 +1,344 @@
+// basic loops
+
+/* Exercise 1:
+Write a for loop that calculates the factorial of 5 (5! = 5 × 4 × 3 × 2 × 1).
+ */
+let f = 1;
+for (let i = 5; i > 1; i--) {
+    f = f * i;
+}
+console.log(f);
+
+/* Exercise 2:
+Modify the loop below to stop when a number greater than 50 is found. Use break here.
+ */
+const numbers = [10, 20, 30, 60, 40, 50];
+for (let i = 0; i < numbers.length; i++) {
+// my code start
+    if (numbers[i] > 50) {
+        break;
+    }
+    console.log(numbers[i]);
+// my code end
+}
+
+/* Exercise 3:
+Given an array of numbers, write a loop that skips all negative numbers and prints only positive numbers. Use continue here.
+const numbers = [10, -5, 20, -8, 30, -2];
+ */
+const numbers1 = [10, -5, 20, -8, 30, -2];
+for (let i = 0; i < numbers1.length; i++) {
+// my code start
+    if (numbers1[i] < 0) {
+        continue;
+    }
+    console.log(numbers1[i]);
+// my code end
+}
+
+/* Exercise 4:
+- Write a for loop that prints Fibonacci numbers up to 10.
+ */
+let x = 0, y = 1;
+for (let i = 0; i < 10; i++) {
+    console.log(x);
+    let sum = x + y;
+    x = y;
+    y = sum;
+}
+
+/* Exercise 5:
+Rewrite the code changing the for loop to while without altering its behavior (the output should stay same).
+ */
+// for (let i = 0; i < 3; i++) {
+//     console.log( `number ${i}!` );
+// }
+let ii = 0;
```

Comment:
Give some informative name of variable. For example you can even write name exerciseFive. Name should be clear